src/MCRun.jl

```diff
@@ -129,31 +149,31 @@ if acceptance rate is >60%/<40%. Acceptance rate is calculated after `n_update`
 each cycle consists of `a` atom, `v` volume and `r` rotation moves.
 Information on actual max. displacement and accepted moves between updates is contained in `mc_state`, see [`MCState`](@ref).  
 """
-function update_max_stepsize!(mc_state::MCState, n_update, a, v, r)
+function update_max_stepsize!(mc_state::MCState, n_update, a, v, r; min_acc = 0.4, max_acc = 0.6)
     #atom moves
```

Comment:
This function now has a minimum and maximum acceptance rate we can define ourselves

Reply:
that's good